pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[build-system]
```

Comment:
I went ahead and swapped from setup.yp to pyproject.toml -- is this still needed if we go this route?

Reply:
I don't think so, no.
We need something that pulls in the obsah dep etc, but otherwise…